### ISSUE TITLE
web ui not found 

### ISSUE THREAD

**Issue by cv-msan** [qdrant/qdrant#5245]
## Steps to Reproduce

1.I downloaded the latest version v1.12.1 for Windows in the releases, and after startup, the log shows' Static content folder for Web UI '/ static' does not exist
![image](https://github.com/user-attachments/assets/1976bfa6-c462-4fcd-aad7-f76e0ecea056)

2.I attempted to pull the code from the qdrant web ui repository and ran npm run build for compilation. The resulting folder 'dist' was renamed 'static' and placed in the root directory of qdrant. After startup, the error disappeared, but when accessing http://localhost:6333/dashboard When the page does not appear, there is an error in browser F12
3.
![image](https://github.com/user-attachments/assets/47e6b5c1-0881-406c-b6e6-ce26155df824)

4.May I ask what I should do to avoid this issue? I don't want to install it in Docker, I just want to use qdrant on Windows system

**Comment by timvisee** [comment]
The web UI is only included in the Docker builds by default. That is to keep the executable plain and simple.

We use this script to grab the necessary files: https://github.com/qdrant/qdrant/blob/master/tools/sync-web-ui.sh

You can do the same. Create a `static` directory in the same directory you start Qdrant from. Grab the dist files from the latest web UI release [here](https://github.com/qdrant/qdrant-web-ui/releases/latest), and put them in the static directory. On restart the web UI should be usable.

Assuming that fixes this, I'm closing this now.

**Comment by cv-msan** [comment]
”You can do the same. Create a static directory in the same directory you start Qdrant from. Grab the dist files from the latest web UI release [here](https://github.com/qdrant/qdrant-web-ui/releases/latest), and put them in the static directory. On restart the web UI should be usable.“Is this operation feasible under the Windows operating system?
I operate in this way, but there are still errors when accessing the browser.
![image](https://github.com/user-attachments/assets/119325e9-491d-4c49-a20c-664efc6720cc)
![image](https://github.com/user-attachments/assets/3a251840-0afb-4f47-abaa-ee378699acff)

**Comment by cv-msan** [comment]
It seems that the requested resource path is incorrect
![image](https://github.com/user-attachments/assets/a783e57b-35e4-42f2-bc3d-be27a5475873)

**Comment by timvisee** [comment]
More specifically, the static directory should be in the current working directory. If you start Qdrant from the directory it's in, that should be fine. If you start it from a different directory, you must change the working directory or move the static files.

Based on the image I'm not sure if the `static` directory is in exactly the same directory as the `qdrant.exe` binary. I'm not sure what the collapsible headers say. But without reading it I'd say it should be like this:

![image](https://github.com/user-attachments/assets/7134db35-a24a-4d4b-8a5e-a7f9d6889891)

The directory structure should be something like this (without the `dist` directory):

```
.
├── qdrant.exe
└── static
    ├── assets
    │   ├── abap-e5f08dc1.js
    │   ├── apex-fc8a734c.js
    │   ├── azcli-c4358445.js
    │   ├── bat-eaac827b.js
    │   ├── bicep-9f8671bb.js
    │   ├── .....
    ├── favicon.ico
    ├── index.html
    ├── logo192.png
    ├── logo512.png
    ├── logo.png
    ├── manifest.json
    ├── openapi.json
    ├── qdrant-web-ui.spdx.json
    └── robots.txt
```

**Comment by cv-msan** [comment]
Yes, my directory structure is consistent with yours, but the web UI is not working properly

**Comment by timvisee** [comment]
How do you start Qdrant?

**Comment by cv-msan** [comment]
Sorry, I misunderstood. To start it, double-click on qdrant.exe with the mouse. The cmd page that appears after startup is as follows.
![image](https://github.com/user-attachments/assets/e74082c6-8964-46fd-ba96-683729060c04)

**Comment by QuanZi123** [comment]
@cv-msan I also have the same problem here. Have you solved it

**Comment by cv-msan** [comment]
> @cv-msan我也在这里遇到了相同的问题。你解决了吗？

No, but it can be done without compilation, so you can directly use VSCode to pull a copy of the front-end source code, and then start npm run start

**Comment by QuanZi123** [comment]
> > @cv-msan我也在这里遇到了相同的问题。你解决了吗？
> 
> No, but it can be done without compilation, so you can directly use VSCode to pull a copy of the front-end source code, and then start npm run start

I use the nginx proxy and it works fine

**Comment by QuanZi123** [comment]
> > @cv-msan我也在这里遇到了相同的问题。你解决了吗？
> 
> No, but it can be done without compilation, so you can directly use VSCode to pull a copy of the front-end source code, and then start npm run start

I took a look at "Dockerfile" and "sync-web-ui.sh" , found that there was a missing step to copy the "openapi.json" file into the static directory:
`
cp "${OPENAPI_FILE}" "${STATIC_DIR}/openapi.json"
`
And if you do that, it should work

**Comment by cv-msan** [comment]
I won't tinker with it now. You can try it on your end and if possible, give them a PR
